modules/core/example/ts/double-spend-example.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Send a transaction from a multi-sig wallet at BitGo.
+ *
+ * Copyright 2019, BitGo, Inc.  All Rights Reserved.
+ */
+import { BitGo, Coin } from 'bitgo';
+const bitgo = new BitGo({ env: 'test' });
+
+const coin = 'tbtc';
+const basecoin = bitgo.coin(coin) as Coin.Btc;
+// TODO: set your access token here
+const accessToken = 'hidden';
+const walletId = 'hidden'; //TODO
+// TODO: set your passphrase here
+const walletPassphrase = 'hidden';
+
+async function main() {
+  bitgo.authenticateWithAccessToken({ accessToken });
+
+  const walletInstance = await basecoin.wallets().get({ id: walletId });
+
+  const newReceiveAddress1 = await walletInstance.createAddress();
+  const newReceiveAddress2 = await walletInstance.createAddress();
+
+  const transaction1 = await walletInstance.send({
```

Comment:
@Xekyo  should we take out the `await` ?

Reply:
Yes